core/src/main/java/org/bitcoinj/evolution/QuorumRotationState.java

```diff
@@ -804,7 +809,7 @@ private ArrayList<ArrayList<SimplifiedMasternodeListEntry>> buildNewQuorumQuarte
                         // we made full "while" loop
                         if (!updated) {
                             // there are not enough MNs, there is nothing we can do here
-                            return new ArrayList<>(nQuorums);
+                            return createNewQuarterQuorumMembers(llmqParameters);
```

Comment:
This was found to be a misalignment with Core 19.  However, during testnet instability this code was triggered and resulted in failure to verify the membership of one or two quorums.